src/dreamer/specialists.py

```
@@ -229,78 +230,167 @@ def iteration_callback(data: Any) -> None:
+                        output_tokens=output_tokens,
+                        duration_ms=duration_ms,
+                        success=specialist_success,
+                        error_class=specialist_error_class,
+                        # denormalized rollups (all 0 on the failure path)
+                        created_observation_count=created_observation_count,
+                        deleted_observation_count=deleted_observation_count,
+                        peer_card_updated=peer_card_updated,
+                        search_tool_calls_count=search_tool_calls_count,
+                    )
+                )
+            except Exception:  # pragma: no cover - telemetry must not raise
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)